bodhi/tests/server/scripts/test_approve_testing.py

```diff
@@ -60,6 +61,8 @@ def test_autokarma_update_meeting_time_requirements_gets_one_comment(self):
         self.assertEqual(
             comment_q[0].text,
             config.get('testing_approval_msg') % update.release.mandatory_days_in_testing)
+        publish.assert_called_once_with(
+            topic='update.requirements_met.stable', msg=ANY)
```

Comment:
I think it might be useful to make sure that more parts of the message are sent correctly, to prevent something like `<Update object at ...>` to be sent in the fedmsg.

Reply:
Added a more specific test. Note that we don't need to worry about `<Update object at ...>` being sent because the `Update` class has a `__json__` method.